spec/models/offline_transaction_charge_spec.rb

```diff
@@ -6,4 +6,8 @@
 
 RSpec.describe OfflineTransactionCharge, type: :model do
 	it_behaves_like 'subtransaction paymentable', :offtrxchrg
+
+	it {
+		is_expected.to have_one(:offsite_payment)
```

Comment:
Make sure to check verify the "through" is properly verified here too (based upon implementing the suggestion in `app/models/offline_transaction_charge.rb`